lib/api/areas.ts

```diff
@@ -4,16 +4,19 @@ import { api } from './'
 
 export interface AreaPatch {
   name: string
+  testException: string
 }
 
 export interface AreaPost {
   name: string
   companyId: string
+  testException: string
```

Comment:
I think a slightly better term might be `testExemption` - it makes the purpose a little more clear and is a distinct term from exceptions as errors, which is something that confused me a little :sweat_smile: